styles.css

```diff
@@ -0,0 +1,129 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  :root {
+    --color-primary: #007bff;
+    --color-base: #fff;
+    --color-light-grey: #ccd5ae;
+    --color-dark-grey: rgba(0,0,0,0.5);
+    --color-dark: #000;
+  }
+  
+  html, body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+  }
+  
+  body {
+    background-color: #f1f2f3;
+  }
+  
+  main {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+    align-content: stretch;
+  }
+  
+  .content--left {
+    margin-right: 48px;
+  }
+  
+  .content--right {
+    max-width: 316px;
+  
+  }
+  
+  ul li { 
+    margin-bottom: 10px; 
+  }
+  
+  
+  .button-signup {
+   display: flex;
+   flex-direction: column;
+   flex-wrap: nowrap;
+   justify-content: space-around;
+   align-items: stretch;
+   align-content: stretch;
+  }
+  
+  
+  .google {
```

Comment:
There's a lot of repetition in google, facebook and github classes. You can move the same code to a generic class, e.g. 
`
  .btn {
    border-radius: 5px;
    margin-bottom: 5px;
    text-align: center;
    text-decoration: none;
    padding: 10px;
  }
`

and then remove these lines from the mentioned classes

Reply:
And where should I add .btn class? Is it <a>

Reply:
You should add 'btn' class that will has styles that all buttons have. Then, you add 'btn' class to all buttons that you have in the page.
After, you can add a second class that only has specific styles (e.g. background color).

HTML will look something like this:
`
<a href="#" class="btn google">Sign up with Google</a>`